README.md

```diff
@@ -33,8 +33,8 @@ Automatically synchronize subscribed customers to Smaily, generate RSS-feed base
 
 ## Requirements
 
-- PHP 5.6 (PHP 7.2+ is recommended)
-- OpenCart 3.0.0.0 to 3.0.3.1
+- PHP 7.3
```

Comment:
Older versions before 3.0.3.5 still support PHP 5.6, thus the module needs to support older versions as well.

It would be best to just remove the PHP version requirement from the README all together. OpenCart requirement already sets the requirements to PHP and database server. No need to duplicate this information here.